src/common/Input/Input.module.css

```diff
@@ -0,0 +1,20 @@
+input {
```

Comment:
you were right the first time, sorry!
```suggestion
.input {
```